app/services/ip_lookup_service.rb

```diff
@@ -1,6 +1,14 @@
 class IpLookupService
   def perform(ip_address)
-    return if ip_address.blank? || !ip_database_available?
+    return if ip_address.blank?
```

Comment:
@mazenkhalil This wouldn't be ideal since it would block the initial requests, triggering the download of the IP database.  We can probably move the logic to an intializer.